tech.json

```diff
@@ -1630,7 +1639,7 @@
         },
         {
           "name": "canCrystalFlash",
-          "requires": [],
+          "requires": [{"noFlashSuit": {}}],
```

Comment:
is using a cf to gain a suit going to be a problem?
i.e. a single strat for elevator CF having `{"noFlashSuit": {}}` and `{"gainFlashSuit": {}}`

Reply:
In a strat like that, it will be fine as long as the `gainFlashSuit` requirement comes after the `noFlashSuit` requirement, which we can be sure to do